SPECS/xcp-ng-generic-lib.spec

```diff
@@ -0,0 +1,57 @@
+Name:           xcp-ng-generic-lib
+Version:        1.0.0
+Release:        1%{?dist}
+Summary:        A library of algorithms, I/O and networking functions...
+License:        GPLv3
+URL:            https://github.com/xcp-ng/xcp-ng-generic-lib
+Source0:        https://github.com/xcp-ng/xcp-ng-generic-lib/archive/v%{version}/%{name}-%{version}.tar.gz
+
+BuildRequires:  cmake3
+BuildRequires:  make
+BuildRequires:  gcc
+
+%description
+A library of algorithms, I/O and networking functions... used by XCP-ng tools or daemons.
+
+%prep
+%autosetup -p1
+
+%build
+mkdir build && cd build
+%cmake3 .. -DCMAKE_INSTALL_PREFIX=%{buildroot}/usr
+make
+
+%install
+cd build
+make install
+
+%files
+%{_libdir}/libxcp-ng-generic.so.1.0
+
+
+%package devel
+Summary:        Development headers for xcp-ng-generic-lib
+Requires:       %{name}%{?_isa} = %{version}-%{release}
+
+%description devel
+A library of algorithms, I/O and networking functions... used by XCP-ng tools or daemons.
+
+This package provides documentation and development headers for xcp-ng-generic-lib.
+
+%files devel
+%doc LICENSE
```

Comment:
I think this should belong to the main package so that the license is always present on the system with the binary, unless you've found some guidelines that suggest otherwise

Reply:
Ok. I don't have an argument against. I have copied this code from https://github.com/xcp-ng-rpms/libempserver/blob/master/SPECS/libempserver.spec#L55

Reply:
A better example: https://src.fedoraproject.org/rpms/libpng/blob/master/f/libpng.spec#_88 => license is with the main package.

I don't know if %license is available in CentOS 7, but %doc is fine I think

Reply:
`%license` is available. So let's use that.